src/main/java/dao/SlotsTimesDao.java

```diff
@@ -0,0 +1,23 @@
+package dao;
+
+import com.google.inject.Inject;
+import domain.SlotsTimes;
+import io.dropwizard.hibernate.AbstractDAO;
+import org.hibernate.SessionFactory;
+
+public class SlotsTimesDao extends AbstractDAO<SlotsTimes> {
+    @Inject
+    public SlotsTimesDao(SessionFactory sessionFactory){
+        super(sessionFactory);
+    }
+
+    public long create(SlotsTimes slotsTimes){
```

Comment:
I don't think we will ever create this programatically, we will use db migrations to populate this table